package.json

```diff
@@ -8,7 +8,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Anthonyzou/react-native-full-screen.git"
+    "url": "git+https://github.com/windastella/react-native-full-screen.git"
```

Comment:
Same to first comment. What is the reason to change the repositories in the package.json? The intentions is to make a new package forked from here?